app/views/__tests__/__snapshots__/Import.spec.js.snap

```diff
@@ -161,14 +161,15 @@ Array [
             }
             use="body1"
           >
-            Visit Google Takeout and export your Location History using following settings: 
+            Visit Google Takeout and export your Location History using the following settings: 
 1. Delivery method: "Add to Drive" 
 2. Frequency: "Export once" 
-3. File type & size: ".zip" and "1GB" 
+3. File type &amp; size: ".zip" and "1GB"
```

Comment:
I was able to confirm that this is showing up as `&amp;` instead of `&` in the app. Looks like the bug is on [this key](https://github.com/tripleblindmarket/covid-safe-paths/pull/648/files#diff-95aa5b48e9899a303ae08d4353789356R18) in en.json